### PR TITLE
Coord translations

### PR DESCRIPTION
**Pull request by rouyeen**

### CODE REVIEW

File: kaos/algorithm/coord_conversion.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    https://www.mathworks.com/help/aeroblks/geodetictogeocentriclatitude.html
+    http://www.jqjacobs.net/astro/geodesy.html
+    http://ccar.colorado.edu/asen5070/handouts/geodeticgeocentric.doc
+    """
+    flattening = 0.00335281068118
+    geod_lat_rad =deg2rad(geod_lat_deg)
+    geoc_lat_rad = atan(((1-flattening)**2)*tan(geod_lat_rad))
+    return rad2deg(geoc_lat_rad)
+
+def lla_to_eci(lat, lon, alt=0, time_posix=946684800):
```
Comment (ZeyadTamimi) [inline kaos/algorithm/coord_conversion.py]: Why the defaults, also doc string the defaults

File: kaos/algorithm/coord_conversion.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+    based on: Department of Defense World Geodetic System 1984"
+              Page 4-4
+              National Imagery and Mapping Agency
+              Last updated June, 2004
+              NIMA TR8350.2
+    """
+
+    # WGS84 ellipsoid constants:
+    a = 6378137
```
Comment (ZeyadTamimi) [inline kaos/algorithm/coord_conversion.py]: Define those as constants above i think: ELLIPSOID_CONSTANT

File: kaos/algorithm/coord_conversion.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    http://ccar.colorado.edu/asen5070/handouts/geodeticgeocentric.doc
+    """
+    flattening = 0.00335281068118
+    geod_lat_rad =deg2rad(geod_lat_deg)
+    geoc_lat_rad = atan(((1-flattening)**2)*tan(geod_lat_rad))
+    return rad2deg(geoc_lat_rad)
+
+def lla_to_eci(lat, lon, alt=0, time_posix=946684800):
+    """
+    Converts geodetic lat,lon,alt, to a Cartesian vector in GCRS frame at the given time.
```
Comment (ZeyadTamimi) [inline kaos/algorithm/coord_conversion.py]: Why is there a new line here

File: kaos/algorithm/coord_conversion.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+              National Imagery and Mapping Agency
+              Last updated June, 2004
+              NIMA TR8350.2
+    """
+
+    # WGS84 ellipsoid constants:
+    a = 6378137
+    e = 8.1819190842622e-2
+    #decimal degrees to radians
+    lat = lat/360*2*pi
```
Comment (ZeyadTamimi) [inline kaos/algorithm/coord_conversion.py]: Why are you doing the conversion yourself, you have functions to this for you :p

File: kaos/algorithm/coord_conversion.py [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+    lon = lon/360*2*pi
+    # intermediate calculation
+    # (prime vertical radius of curvature)
+    N = a / sqrt(1 - pow(e,2) * pow(sin(lat),2))
+    # results:
+    x = (N+alt) * cos(lat) * cos(lon)
+    y = (N+alt) * cos(lat) * sin(lon)
+    z = ((1-pow(e,2)) * N + alt) * sin(lat)
+
+    return (x,y,z)
```
Comment (ZeyadTamimi) [inline kaos/algorithm/coord_conversion.py]: Vector3D?

File: kaos/algorithm/coord_conversion.py (hunk 1 above)
Comment (rouyeen) [reply kaos/algorithm/coord_conversion.py]: Defaults no more!

**Pull request merged**